bokeh/charts/attributes.py

```diff
@@ -107,6 +110,9 @@ def __init__(self, columns=None, df=None, iterable=None, default=None,
 
         super(AttrSpec, self).__init__(**properties)
 
+        if self.default is None and self.iterable is not None:
+            self.default = next(copy(iter(self.iterable)))
+
```

Comment:
You don't need to copy here, do you? This breaks in 2.7.. if you remove this tests should pass..